src/delete.js

```diff
@@ -0,0 +1,31 @@
+import component from './displayHtml.js';
+import { tasks } from './taskWrk.js';
+
+const deleteTask = () => {
+  const clearBtn = document.getElementById('button');
+  clearBtn.addEventListener('click', () => {
+    const result = tasks.filter((x) => x.complete === false);
+    tasks.splice(0);
+    tasks.push(...result);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    component();
+  });
+};
+
+const deleteTrash = () => {
+  const trash = document.querySelectorAll('#trash');
+  trash.forEach((e) => {
+    e.addEventListener('click', () => {
+      const index = e.parentNode.parentNode.id;
+      if (tasks[index].complete === true) {
+        e.parentNode.parentNode.remove();
+        tasks.splice(index, 1);
+        localStorage.setItem('tasks', JSON.stringify(tasks));
+        component();
+        window.location.reload();
+      }
+    });
+  });
+};
```

Comment:
- [ ] you can append the trash to make it work, see below the trash icon is not working 👇

https://user-images.githubusercontent.com/61562626/126048941-7d16d806-caba-4673-84ea-6281ecd1c2fd.mp4

.

Reply:
Thank you! I thought that only checked tasks should be deleted, but I saw the video and I was wrong. So I changed as you requested!